lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -345,21 +410,82 @@ def _discard_tunnel_flows(self, precedence:int, i_teid:int,
         flows.delete_flow(self._datapath, self.tbl_num, match,
                           priority=priority + 1)
 
-        # Forward flow for LOCAL port
-        if not ue_ip_adr:
-            self.logger.error("ue_ip_address is None")
-            return
+        if ip_flow_dl.set_params:
+            self._discard_tunnel_ip_flow_dl(ip_flow_dl)
         else:
-            ip_match_out = get_ue_ip_match_args(ue_ip_adr, Direction.IN)
-            match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
-                               in_port=self._uplink_port, **ip_match_out)
-
-            flows.delete_flow(self._datapath, self.tbl_num, match,
-                              priority=priority +1)
-
-            match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
-                               in_port=self.config.mtr_port, **ip_match_out)
-
-            flows.delete_flow(self._datapath, self.tbl_num, match,
-                              priority=priority + 1)
+            # Forward flow for LOCAL port
+            if not ue_ip_adr:
+                self.logger.error("ue_ip_address is None")
+                return
+            else:
+                ip_match_out = get_ue_ip_match_args(ue_ip_adr, Direction.IN)
+                match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
+                                   in_port=self._uplink_port, **ip_match_out)
+
+                flows.delete_flow(self._datapath, self.tbl_num, match,
+                                  priority=priority +1)
+
+                match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
+                                   in_port=self.config.mtr_port, **ip_match_out)
+
+                flows.delete_flow(self._datapath, self.tbl_num, match,
+                                  priority=priority + 1)
+
+    def _discard_tunnel_ip_flow_dl(self, ip_flow_dl:IPFlowDL):
```

Comment:
Here and a few other places can you fix spacing between the typehint, its
```
def _discard_tunnel_ip_flow_dl(self, ip_flow_dl: IPFlowDL):
```

Reply:
Changes done